main.go

```diff
@@ -454,8 +454,9 @@ Features detected:
 
 	fmt.Printf("\n")
 
```

Comment:
You should probably short this list of strings (functionNamespaces)

https://gobyexample.com/sorting

Reply:
The namespaces are now sorted.